web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Links/page/LinksForm/index.tsx

```diff
@@ -81,6 +82,14 @@ export const LinksForm = (props: LinksFormProps) => {
     },
   });
 
+  useEffect(() => {
```

Comment:
@andy-t-wang What is this for? 
Let's left a comment here

Reply:
the form doesn't update with the new values when we change the selector